decidim-participatory_processes/config/locales/en.yml

```
@@ -306,6 +306,7 @@ en:
           error: There was a problem creating a new participatory process group.
           success: Participatory process group successfully created.
       titles:
+        new_participatory_process_type: New participatory process type
```

For consistency with the view:
```suggestion
        new_participatory_process_type: New process type
```